src/NuGet.Core/NuGet.ProjectModel/DependencyGraphSpec.cs

```diff
@@ -247,6 +237,74 @@ public void Save(string path)
             }
         }
 
+        internal static DependencyGraphSpec Load(string path, bool useImprovedReader)
```

Comment:
The only place `useImprovedReader` is set to false is in tests. It was a good way to check for regressions during development, but since the production code only ever uses `true`, should it be deleted now? the `false` codepath is now dead code.